README.md

```diff
@@ -66,209 +64,282 @@ of `fetcher` and rerenders the component.
 Note that `fetcher` can be any asynchronous function, so you can use your favourite data-fetching
 library to handle that part.
 
----
+Check out [swr.now.sh](https://swr.now.sh) for more demos of SWR.
+
+## Usage
 
-- API
-  - [`useSWR`](#useswr)
-  - [`SWRConfig`](#swrconfig)
-  - [`mutate`](#mutate)
-  - [`trigger`](#trigger)
-- Examples
-  - [Suspense Mode](#suspense-mode)
-  - [Subscription (e.g.: socket.io)](#subscription-eg-socketio)
-  - [Dependent Fetching](#dependent-fetching)
+Inside your React project directory, run the following:
 
-## API
+```
+yarn add swr
+```
 
-### `useSWR`
+Or with npm:
+
+```
+npm install swr
+```
+
+### API
 
 ```js
-const {
-  data,                                    // data for the given key (or undefined)
-  error,                                   // error (or undefined)
-  isValidating,                            // if the request is loading
-  revalidate                               // function to trigger a validate manually
-} = useSWR(
-  key,                                     // a unique key for the data (or a function, see below)
-  fetcher,                                 // Promise returning function to load your data
-  swrOptions? = {
-    suspense: false,                       // enabled React Suspense mode
-    revalidateOnFocus: true,               // auto revalidate when window gets focused
-    refreshWhenHidden: false,              // refresh while the window is invisible
-    shouldRetryOnError: true,              // retry when fetcher has an error
-    refreshInterval: 0,                    // polling interval (disabled by default)
-    errorRetryInterval: 5000,              // error retry interval (10s on slow network)
-    focusThrottleInterval: 5000,           // keep focus revalidate requests in a time window
-    dedupingInterval: 2000,                // deduping requests
-    loadingTimeout: 3000,                  // timeout for triggering the onLoadingSlow event
-
-    onLoadingSlow,                         // event handlers
-    onSuccess,
-    onError,
-    onErrorRetry,
-
-    fetcher                                // default fetcher function
-  }
-)
+const { data, error, isValidating, revalidate } = useSWR(key, fetcher, options)
 ```
 
-#### `key` as a function
+Example usage:
```

Comment:
This seems out of place to me in the API section, can we move it down to the examples section under `### Local Configuration`?

Reply:
Hmmm actually I think this can be removed because we have the introduction section already.